thirdparty/github.go

```diff
@@ -2022,6 +2022,13 @@ func UpdateCheckRun(ctx context.Context, owner, repo, uiBase string, checkRunID
 	))
 	defer span.End()
 
+	if output == nil {
+		output = &github.CheckRunOutput{
+			Title:   utility.ToStringPtr("Task restarted with no output"),
+			Summary: utility.ToStringPtr(fmt.Sprintf("Task '%s' at execution %d was completed", task.DisplayName, task.Execution)),
+		}
+	}
+
```

Comment:
I think we should test this on staging. If you already did, can you please link it?

Reply:
yes it's the same check run link! (the execution starts at 0 so the summary is wrong but I added a +1 (but did not test that)